.editorconfig

```diff
@@ -0,0 +1,14 @@
+root = true
```

Comment:
meaning this editorconfig file is located in the root of the project, I guess it would traverse from one folder up until it find a config file that claim to be root and stop